server/include/CFileSystem.h

```diff
@@ -1,67 +1,68 @@
-#ifndef CFILESYSTEM_H

```

Comment:
What the heck did git do to this file?

Reply:
@LoneBoco I assume we haven't touched this file ever since it was migrated from SVN to GIT, and the whole file was transformed from Windows newlines to Unix newlines...

Reply:
The change in this file is minor though; `getDirByExtension` was added